packages/components/src/utils/getBreadcrumbFromSiteMap.ts

```diff
@@ -7,11 +7,16 @@ export const getBreadcrumbFromSiteMap = (
   sitemap: IsomerSitemap,
   permalink: string[],
 ): BreadcrumbProps => {
-  const breadcrumb = []
+  const breadcrumb = [
+    {
+      title: "Home",
+      url: "/",
+    },
+  ]
   let node = sitemap
   let currentPath = ""
 
-  for (const pathSegment of permalink) {
+  for (const pathSegment of permalink.slice(0, -1)) {
```

Comment:
hmm this one abit error prone.
shall we create a util function called `removeTrailingSlash` that checks for the existence of it and then slices it? this way we are more resilient to errors if upstream callers dont put the slash

Reply:
oh hmm the `slice` isn't to check for a trailing slash tho, it's to remove the last item in the permalink which is the current page. is it still error prone this way?

Reply:
hmm if that is the case what happens if i am in the top level ah

Reply:
Just 'home' would be displayed